### PR TITLE
[LIB 315] Add GCM devices and GCM messages to the python LIB

### PR DESCRIPTION
**Pull request by opalczynski**

### CODE REVIEW

**Comment by opalczynski** [comment]
@dancio @Zhebr Some feedback will be nice :) Then I can start developing the APNS pushes;

File: syncano/models/push_notification.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                'path': '/push_notifications/gcm/devices/{registration_id}/',
+            },
+            'list': {
+                'methods': ['get'],
+                'path': '/push_notifications/gcm/devices/',
+            }
+        }
+
+    def is_new(self):
+        if self.created_at is None:
```
Comment (Zhebr) [inline syncano/models/push_notification.py]: maybe just ```return self.created_at is None``` ?

File: syncano/models/fields.py [hunk 2]
```diff
@@ -579,6 +579,18 @@ def to_native(self, value):
...
+        if value is None:
+            return
+
+        if not isinstance(value, six.string_types):
+            value.update({
+                'environment': PUSH_ENV,
+            })
+            value = json.dumps(value)
+        return value
+
```
Comment (Zhebr) [inline syncano/models/fields.py]: one newline more here

**Comment by Zhebr** [comment]
nice!

**Pull request merged**